content/academy/data_collection_basics/api_scraping.md

```diff
@@ -90,20 +76,39 @@ Since our request doesn't have any body/payload, we just need to analyze the URL
 
 Understanding an API's various configurations helps with creating a game-plan on how to best scrape it, as many of the parameters can be utilized for easy pagination, or easy data-filtering. Additionally, these values can be mapped to a scraper's configuration options, which overall makes the scraper more versatile.
 
-Based on our observations, we can modify the URL and utilize the "`limit`" option to return 200 songs in one request, instead of just 20:
+Let's say we want to receive all of the user's tracks in one request. Based on our observations of the endpoint's different parameters, we can modify the URL and utilize the "`limit`" option to return more than just twenty songs:
 
 ```
-https://api-v2.soundcloud.com/users/141707/tracks?client_id=zdUqm51WRIAByd0lVLntcaWRKzuEIB4X&limit=200
+https://api-v2.soundcloud.com/users/141707/tracks?client_id=zdUqm51WRIAByd0lVLntcaWRKzuEIB4X&limit=99999
 ```
 
+By using the ridiculously large number of `99999`, we ensure that all of the user's tracks will be captured in this single request.
+
 # [](#challenges) Dealing with headers, cookies, and tokens
 
 Unfortunately, most APIs will require a valid cookie to be included in the "`cookie`" field within a request's headers in order to be authorized. Other APIs may require special tokens, or other data that validates the request.
 
-Luckily, there are ways to retrive and set cookies for requests prior to sending them, which will be covered more in-depth future Scraping Academy modules. The most important things to know at the moment are:
+Luckily, there are ways to retrive and set cookies for requests prior to sending them, which will be covered more in-depth within future Scraping Academy modules. The most important things to know at the moment are:
 
 1. For sites that heavily rely on cookies for user-verification and request authorization, certain generic requests (such as to the website's main page, or to the target page) will return back a (or multiple) "`set-cookie`" header(s).
-2. The `set-cookie` response header(s) can be parsed and used as the `cookie` header in the headers of a request. A great package for parsing these values from a response's headers is <a href="https://www.npmjs.com/package/set-cookie-parser">`set-cookie-parser`</a>
+2. The `set-cookie` response header(s) can be parsed and used as the `cookie` header in the headers of a request. A great package for parsing these values from a response's headers is [`set-cookie-parser`](https://www.npmjs.com/package/set-cookie-parser). With this package, cookies can be parsed from headers like so:
+
+```JavaScript
+const axios = require('axios');
+const setCookieParser = require('set-cookie-parser'); // import the set-cookie-parser module
```

Comment:
Let's not do this now, but we should think about promoting the SDK in the code examples and courses. Maybe we could add some helpers that make working with cookies easier. Ideas welcome.